lib/config/fileconf.go

```diff
@@ -1538,6 +1540,13 @@ type AccessGraphAWSSync struct {
 	ExternalID string `yaml:"external_id,omitempty"`
 }
 
+type AccessGraphAzureSync struct {
+	Regions        []string `yaml:"regions,omitempty"`
+	SubscriptionID string   `yaml:"subscription_id,omitempty"`
+	UMIClientID    string   `yaml:"umi_client_id,omitempty"`
```

Comment:
Do we need this?

Reply:
Not for now, possibly not ever. We can get default creds from the VM running Discovery.